backend/capellacollab/users/models.py

```diff
@@ -34,14 +41,14 @@ class PatchUserRoleRequest(BaseModel):
 class DatabaseUser(Base):
     __tablename__ = "users"
 
-    id = Column(Integer, primary_key=True, index=True)
-    name = Column(String, unique=True, index=True)
-    role = Column(Enum(Role))
-    projects = relationship(
+    id: int = Column(Integer, primary_key=True, index=True)
+    name: str = Column(String, unique=True, index=True)
+    role: Role = Column(Enum(Role))
+    projects: list["ProjectUserAssociation"] = relationship(
```

Comment:
Is SQLAlchemy not able to abstract over ProjectUserAssociation? It's only an a link for the n:m relationship between projects and users.

Reply:
As far as I know, it doesn’t, and since `ProjectUserAssociation` has properties we want to access, it’s better to keep it like this.